test/unit_Zonotope.jl

```diff
@@ -90,6 +90,11 @@ for N in [Float64, Rational{Int}, Float32]
     @test Z5.center == N[0, 1]
     @test Z5.generators == N[0.5 0.5 0.5 0; -0.5 0.5 0 0.5]
 
+    # in-place scale
+    Z5aux = copy(Z3)
+    scale!(0.5, Z3)
+    @test Z5 ≈ Z5aux
```

Comment:
I do not understand this test. Is there a typo?

Reply:
yes, thanks